app/src/main/java/com/main/mealplanner/dto/RecipeHeader.kt

```diff
@@ -1,7 +1,5 @@
 package com.main.mealplanner.dto
 
-import androidx.room.Entity
-import androidx.room.PrimaryKey
```

Comment:
Removed the unused imports so the code looks cleaner.
Source:
https://kotlinlang.org/docs/coding-conventions.html#verify-that-your-code-follows-the-style-guide

Reply:
Control Alt S does this automatically.